tools/lsp/preview.rs

```diff
@@ -83,6 +88,140 @@ fn set_design_mode(enable: bool) {
     configure_design_mode(enable);
 }
 
+fn follow_child(element: &ElementRc) -> ElementRc {
```

Comment:
what does that do?  Deserve a comment.   I'm not getting what "child" is in this context.  It looks like it return the repeated element root or itself.